h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -63,7 +63,7 @@
  */
 public class EasyPredictModelWrapper implements Serializable {
   // These private members are read-only after the constructor.
-  public final GenModel m;
+  private final GenModel m;
```

Comment:
I wouldn't change this - it has no actual benefit and can break code for users => not worth it.

Reply:
True, thank you, I didn't think of that 🤦 So it will be just about refactoring on the new method if at least this worth it